bgpd/bgp_mplsvpn.c

```diff
@@ -631,108 +642,97 @@ static bool sid_exist(struct bgp *bgp, const struct in6_addr *sid)
 	return false;
 }
 
-/*
- * This function generates a new SID based on bgp->srv6_locator_chunks and
- * index. The locator and generated SID are stored in arguments sid_locator
- * and sid, respectively.
+/**
+ * Return the SRv6 SID value obtained composing the LOCATOR and FUNCTION.
```

Comment:
nit ... /obtained composing/ -> /obtained by composing/ ??

Reply:
Hi @riw777 Many thanks for the review.
I fixed the mistake.